pipeline/preprocessing.py

```diff
@@ -217,41 +220,40 @@ def ANTS_rigid_body_trans(b0_nii, reference=None):
     return (transformed_file, omat_file)
 
 
-def pre_process(lock, subject, reference_list):
+def pre_process(input_file, reference_list, b0_threshold= None, which_bse= '--avg'):
 
```

Comment:
The default arguments: `b0_threshold= None, which_bse= '--avg'` are introduced to facilitate future modification.

Reply:
Thank you Tashrif